cmake/linker_script/common/common-rom.cmake

```diff
@@ -213,3 +213,8 @@ endif()
 if(CONFIG_USBD_MSC_CLASS)
   zephyr_iterable_section(NAME usbd_msc_lun KVMA RAM_REGION GROUP RODATA_REGION SUBALIGN 4)
 endif()
+
+if(CONFIG_ZBUS)
+  zephyr_iterable_section(NAME zbus_channel KVMA RAM_REGION GROUP RODATA_REGION SUBALIGN 4)
```

Comment:
You already have this in the `common-ram.cmake` file:
```
if(CONFIG_ZBUS)
  zephyr_iterable_section(NAME zbus_channel GROUP DATA_REGION ${XIP_ALIGN_WITH_INPUT} SUBALIGN 4)
  zephyr_iterable_section(NAME zbus_observer GROUP DATA_REGION ${XIP_ALIGN_WITH_INPUT} SUBALIGN 4)
endif()
```

Reply:
@nordicjm I removed these from the `common-ram.cmake`, leaving them on the `common-rom.cmake`, but I still cannot compile samples and tests for Xtensa boards.

Reply:
Solved! I created a `common-rom-zbus.ld` file and added that to the `common-rom` directory. However, some architectures use the LD files directly. So, when I moved that to the `common-rom-misc.ld` file, the problem disappeared.

Reply:
Thank you @fabiobaltieri and @nordicjm.